locales/es/pokemon.json

```diff
@@ -1086,15 +1086,15 @@
   "Phantump": "Phantump",
   "Trevenant": "Trevenant",
   "Pumpkaboo": "Pumpkaboo",
-  "Pumpkaboo (Average)": "",
-  "Pumpkaboo (Small)": "",
-  "Pumpkaboo (Large)": "",
-  "Pumpkaboo (Super Size)": "",
+  "Pumpkaboo (Average)": "[[Pumpkaboo]] (Normal)",
+  "Pumpkaboo (Small)": "[[Pumpkaboo]] (Pequeño)",
+  "Pumpkaboo (Large)": "[[Pumpkaboo]] (Grande)",
+  "Pumpkaboo (Super Size)": "[[Pumpkaboo]] (Supergrande)",
   "Gourgeist": "Gourgeist",
-  "Gourgeist (Average)": "",
-  "Gourgeist (Small)": "",
-  "Gourgeist (Large)": "",
-  "Gourgeist (Super Size)": "",
+  "Gourgeist (Average)": "[[Gourgeist]] (Normal)",
+  "Gourgeist (Small)": "[[Gourgeist]] (Pequeño)",
+  "Gourgeist (Large)": "[[Goirgeist]] (Grande)",
+  "Gourgeist (Super Size)": "[[Gourgeist]] (Extra grande)",
```

Comment:
Why is pumpkaboo `Supergrande` but Gourgeist is `Extra grande`? I'd expect them to be the same